pgrx/src/spi/client.rs

```diff
@@ -76,29 +77,6 @@ impl SpiClient {
         query.execute(self, limit, args)
     }
 
-    pub(super) fn prepare_tuple_table(
-        &self,
-        status_code: i32,
-    ) -> std::result::Result<SpiTupleTable, SpiError> {
-        Ok(SpiTupleTable {
-            status_code: Spi::check_status(status_code)?,
-            // SAFETY: no concurrent access
-            table: unsafe { pg_sys::SPI_tuptable.as_mut()},
-            #[cfg(any(feature = "pg11", feature = "pg12"))]
-            size: unsafe { pg_sys::SPI_processed as usize },
-            #[cfg(not(any(feature = "pg11", feature = "pg12")))]
-            // SAFETY: no concurrent access
-            size: unsafe {
-                if pg_sys::SPI_tuptable.is_null() {
-                    pg_sys::SPI_processed as usize
-                } else {
-                    (*pg_sys::SPI_tuptable).numvals as usize
-                }
-            },
-            current: -1,
-        })
-    }
-
```

Comment:
This code was moved over into the new `SpiTupleTable::wrap()` function.  Doing so allows us to set its members private, which is much easier to reason about.